CodeStarterCamp_Week3/step1.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  step1.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by MARY on 2023/03/07.
+//
+
+import Foundation
+
+struct Person {
+    let name: String
+    var money: Int
+    func order(_ coffee: Coffee, at coffeshop: CoffeeShop) {
+        print("주문: \(coffee) (\(name))")
+        coffeshop.make(coffee)
+    }
+    init(name: String, money: Int) {
```

Comment:
메서드와 메서드 사이 역시 한번 개행해줍니다 ㅎㅎ 
```suggestion
    }
    
    init(name: String, money: Int) {
```